src/vue/backend/src/templates/AgentsSettings.vue

```diff
@@ -27,7 +27,7 @@
             <p>Default Number of Posts Description of the Default Number of Posts setting in IMPress Agents. Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
         </idx-block>
         <idx-form-group>
-            <idx-form-label customClass="form-content__label" for="number-of-posts">Default Number of Posts</idx-form-label>
+            <idx-form-label customClass="form-content__label" target="number-of-posts">Default Number of Posts</idx-form-label>
```

Comment:
The targets and labels will need to be prefixed with the `idxStrap` prefix. Same for all target/id pairs

Reply:
The `id` isn't being prefixed with `idx-wp-` so leaving the `idxStrap` prefix off makes them match.
![Screen Shot 2021-03-24 at 7 41 09 AM](https://user-images.githubusercontent.com/1541291/112329488-790e3000-8c74-11eb-91a8-6f7046392361.png)

Reply:
I think I get what you're saying here, I should add the prefix to both the label and its target...